.travis.yml

```diff
@@ -214,10 +213,27 @@ script:
   #
   # Enable COREYX (swapped)
   #
-  #- restore_configs
-  #- opt_enable COREYX
-  #- build_marlin
+  - restore_configs
+  - opt_enable COREYX
+  - build_marlin
+  #
+  # Test many less common options
   #
+  - restore_configs
+  - opt_set_adv FAN_MIN_PWM 50
+  - opt_set_adv FAN_KICKSTART_TIME 100
+  - opt_set_adv XY_FREQUENCY_LIMIT  15
+  - opt_enable_adv SHOW_TEMP_ADC_VALUES HOME_Y_BEFORE_X EMERGENCY_PARSER
+  - opt_enable_adv ABORT_ON_ENDSTOP_HIT_FEATURE_ENABLED ADVANCED_OK
+  - opt_enable_adv VOLUMETRIC_DEFAULT_ON NO_WORKSPACE_OFFSETS ACTION_ON_KILL
+  - opt_enable_adv EXTRA_FAN_SPEED FWERETRACT Z_DUAL_STEPPER_DRIVERS Z_DUAL_ENDSTOPS
+  - opt_enable_adv MENU_ADDAUTOSTART SDCARD_SORT_ALPHA
+  - opt_enable ULTRA_LCD FILAMENT_LCD_DISPLAY FILAMENT_WIDTH_SENSOR
+  - opt_enable REPRAP_DISCOUNT_SMART_CONTROLLER
+  - opt_enable ENDSTOP_INTERRUPTS_FEATURE FAN_SOFT_PWM SDSUPPORT
+  - opt_enable USE_XMIN_PLUG USE_XMAX_PLUG USE_YMIN_PLUG
+  - opt_enable USE_YMAX_PLUG USE_ZMIN_PLUG USE_ZMAX_PLUG
+  - build_marlin
```

Comment:
There's no reason you can't combine the COREYX test with the rest of these options.